package.json

```diff
@@ -25,6 +25,7 @@
     "hubot": ">= 2.6.0 < 3.0.0",
     "hubot-scripts": ">= 2.5.0 < 3.0.0",
     "hubot-irc": "git+https://github.com/sshirokov/hubot-irc",
+    "hubot-test-helper": ">= 0.0.3"
```

Comment:
This should be in `devDependencies` since there's no reason for it to be deployed